frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/OptionTabs.test.tsx

```diff
@@ -114,26 +114,23 @@ describe('EditOptions', () => {
     ).toBeInTheDocument();
   });
 
-  it('should render EditOptionChoice when featureFlag is enabled', async () => {
+  it('should render ManualOptionsEditor when manual options are set and featureFlag is enabled', async () => {
```

Comment:
I know I commented this same test earlier, but I still think it can be concretized;
```suggestion
  it('should render the option label when manual options are set and featureFlag is enabled', async () => {
```

Or maybe adapt what is expected to find in the test. Would it be better looking for the preview title for the manual options: `ux_editor.modal_properties_code_list_custom_list` and adapting the test case accordingly?

Reply:
I see you fixed this, great 👏 But now both the title and the expect was changed to still not match 🫣 
Since you changed the expect, maybe something like this would make more sense: `should render the preview title for manual options when manual options are set and featureFlag is enabled`?